projectread/settings.py

```diff
@@ -83,12 +83,15 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": "postgres",
+        "USER": "postgres",
+        "PASSWORD": "postgres",
```

Comment:
Do we wanna move the password into .env? Also, are we planning to change the password soon?

Reply:
Should be fine :) It's just the development database and it won't (or at least shouldn't) have any sensitive data